<?xml version="1.0"?>

<launch>
  <arg name="ns" default="vigir_step_controller" />
  <group ns="$(arg ns)">
    <!-- start walk controller -->
    <node name="step_controller" pkg="vigir_step_control" type="step_controller_node" respawn="true" output="screen" />
    <param name="step_controller_plugin" value="step_controller_test_plugin" />
    <param name="step_plan_msg_plugin" value="step_plan_msg_plugin" />
  </group>
</launch>
